public/javascripts/SVLabel/src/SVLabel/task/TaskContainer.js

```diff
@@ -125,6 +125,11 @@ function TaskContainer (navigationModel, neighborhoodModel, streetViewService, s
         }));
         paths = null;
 
+        // updates the segments that the user has already explored
+        self.update();
+        // updates the next group of segments that the user will explore
```

Comment:
So the next task is a single "street edge", which can technically be made up of multiple line segments. But we typically think in terms of streets, so It might be less confusing for this comment to say "Renders the next street that the user will explore." or something. Lmk if you think that description is not accurate though.